src/block_service/test/fds_service_test.cpp

```diff
@@ -774,7 +774,7 @@ TEST_F(FDSClientTest, test_concurrent_upload_download)
         for (unsigned int i = 0; i < total_files; ++i) {
             block_file_ptr p = block_files[i];
             dsn::task_ptr t =
-                p->download(download_request{filenames[i] + ".b", 0, -1},
+                p->download(download_request{filenames[i] + ".b", 0, -1, ""},
```

Comment:
passing `empty` will not cover your added code, you should add new case to test this pr

Reply:
+1